### PR TITLE
Pic 306 populate test data

### PR DESCRIPTION
**Pull request by jhevans**
The task was to create a script will allow us to populate dev and preprod with data that will match with corresponding data in delius test so that we can perform full integration testing. This is a temporary measure until such a time as the  matcher starts populating court-case-service properly.  The script I've created does this but there are a number of problems which have come to light:

- Kotlin is not very good at scripting (at this time), in order to resolve dependencies easily I've used the kscript extensions but this only works with Java 8 which is annoying and would make me highly dubious about adopting this method for routine scripting tasks in future.
- In retrospect this task might be better off implemented in prepare-a-case as part of an integration test pipeline. This would remove the need for a cronjob to make sure there's always some usable data for a given day, we could just run the integration tests and have them bootstrap their own data. If we need demo data - just run the integration suite.

In summary, this does the job well enough but may become redundant pretty quickly.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -159,4 +164,17 @@ dependencies {
...
+    implementation "org.jetbrains.kotlin:kotlin-script-runtime:1.3.71"
+    implementation "com.github.kittinunf.fuel:fuel:2.2.1"
+
 }
+
+
+task("createTestData") {
+
+    doLast {
+        println("Hello from foo task")
```
[the remark this answers is not in the document]
Comment (jhevans) [reply build.gradle]: nope! removed
